.github/workflows/data-update.yml

```diff
@@ -0,0 +1,23 @@
+name: Axe data update
+on:
+  schedule:
+    # Run daily, 5:00 UTC - an hour before the WAI site checks for updates
+    - cron: '0 5 * * *'
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v2
+      - uses: c-hive/gha-yarn-cache@v2
+      ## Make sure we have the latest version of axe-core
+      - run: yarn add axe-core@next
```

Comment:
Do we want `next`? Latest develop branch changes aren't guaranteed to be QA'd yet so this could introduce bad results if there was a bug in one of the changes